installscripts/wizard/scenarios/support/jazz_common.py

```diff
@@ -4,11 +4,35 @@
 import subprocess
 
 # Global variables
-HOME_FOLDER = os.path.expanduser("~")
-INSTALL_SCRIPT_FOLDER = HOME_FOLDER + "/jazz-installer/installscripts/"
+HOME_FOLDER = "~"
 
-TERRAFORM_FOLDER_PATH = INSTALL_SCRIPT_FOLDER + "/jazz-terraform-unix-noinstances/"
-TFVARS_FILE = TERRAFORM_FOLDER_PATH + "terraform.tfvars"
+
+def get_installer_root():
+    return os.environ['JAZZ_INSTALLER_ROOT']
+
+
+def get_script_folder():
+    return get_installer_root() + "/installscripts/"
+
+
+def get_terraform_folder():
+    return get_script_folder() + "/jazz-terraform-unix-noinstances/"
+
+
+def get_tfvars_file():
+    return get_terraform_folder() + "terraform.tfvars"
+
+
+def get_docker_path():
+    return get_script_folder() + "dockerfiles/"
+
+
+def get_jenkins_pem():
+    return get_docker_path() + "jenkins/jenkinskey.pem"
+
+
+def get_atlassian_tools_path():
+    return get_installer_root() + "jazz_tmp/atlassian-cli-6.7.1/"
```

Comment:
@bleggett  To avoid path error, Please make this line to - 
return get_installer_root() + "/jazz_tmp/atlassian-cli-6.7.1/"